logstash-input-sqlite.gemspec

```diff
@@ -20,11 +20,12 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "input" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", "~> 2.0.0.snapshot"
+  s.add_runtime_dependency "logstash-core", "2.0.0.snapshot2"
```

Comment:
You don't need to have a `:path` defined, because the devutils and the logstash-core gems exist on rubygems.